### PR TITLE
make HA language more explicit

### LINKED ISSUE #1956: Update language around high availability services
(From a Slack conversation initiated by Parth)

https://circleci.com/docs/2.0/high-availability/

***NOTE: Your CircleCI license must include support for high-availability in order to use this configuration. Please contact your account manager if you have questions.***

Can we alter that NOTE a bit, it's causing a bit of confusion that I think can be  avoided. It makes it seem like they can do a HA installation without first having to contact us.

Something like:

***NOTE: Your CircleCI license must include support for high-availability in order to use this configuration. Please contact your Customer Success Manager if you'd like to get started on a High Availability Installation on CircleCI 2.0***
circleci.com
Adding External Database Hosts for High Availability - CircleCI
Configuring High Availability for CircleCI 2.0

### PR DESCRIPTION
**Pull request by smart-alek**
Fixes #1956. Also makes "Note" capitalization consistent and removes dash from "high availability" phrase.

### CODE REVIEW

File: jekyll/_cci2/high-availability.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ order: 20
 description: "Configuring High Availability for CircleCI 2.0"
 ---
 
-**NOTE: Your CircleCI license must include support for high-availability in order to use this configuration. Please contact your account manager if you have questions.**
+**Note: To use this configuration, your CircleCI license must include support for high availability. Please contact your Customer Success Manager if you'd like to set up a High Availability Installation on CircleCI 2.0.**
```
Comment (michelle-luna) [inline jekyll/_cci2/high-availability.md]: I would add a section for this and reword it to make it stronger:

##Prerequisites

Before you configure an existing CircleCI installation for high availability, you must update your license by contacting the Customer Success Manager on your account or by opening a [support ticket]( https://support.circleci.com/hc/en-us/requests/new). Configuring an existing CircleCI installation for HA without first updating the license through the CircleCI customer success team is **not** supported.

Comment (ndintenfass) [inline jekyll/_cci2/high-availability.md]: I've purposefully avoided "CSM" in docs, as many people for whom this is relevant don't yet have one and are dealing with AE/SE.

Comment (smart-alek) [reply jekyll/_cci2/high-availability.md]: Makes sense. But if we block HA configuration with CSMs, we should be explicit about that relationship. @michelle-luna thoughts?
